index.html

```diff
@@ -150,6 +160,7 @@ <h2>Multi-post <br>Story</h2>
       </div>
     </section>
     <section class="about-sect">
```

Comment:
- [ ] As the previous reviewer mentioned, the left side of the ```About``` section should  be smaller but also the right side should be wider, please implement this in your design and consider using the same font sizes and margin/padding, so your design is similar to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220603_212717](https://user-images.githubusercontent.com/25227022/171947849-4ea5eab8-8b5a-4dc1-91a5-b6f00cd7fd01.png)|![](https://user-images.githubusercontent.com/33752735/171758458-4cd65cb6-7c46-4c3b-9345-53ca29c8169b.png)|